frontend/awx/resources/templates/WorkflowVisualizer/components/CustomLabel.tsx

```diff
@@ -0,0 +1,65 @@
+import { useSize } from '@patternfly/react-topology';
+import { CustomLabelProps, EdgeStatus } from '../types';
+import { LegacyRef } from 'react';
+
+const EdgeStatusColors = {
+  [EdgeStatus.danger]: 'var(--pf-v5-global--danger-color--100)',
```

Comment:
We should re-use established color variables from `pfcolors.tsx` when possible.

Reply:
Fixed in follow-up commit [13033a5](https://github.com/ansible/ansible-ui/pull/1225/commits/13033a505ac09e53791ce562319a9af03ef3e854)